starboard/elf_loader/exported_symbols.cc

```diff
@@ -230,6 +231,7 @@ ExportedSymbols::ExportedSymbols() {
 
   // POSIX APIs
   REGISTER_SYMBOL(aligned_alloc);
+  REGISTER_SYMBOL(atexit);
```

Comment:
Is this related to the rest of the pr ? I could we we may need `atexit` but it's not clear from here as to how those two are linked.